tests/test_admin.py

```diff
@@ -809,6 +861,119 @@ def test_menuitem_add_view_redirects_on_save_add_another(self):
         )
 
 
+class MenuItemAdminPreviewViewTestCase(CMSTestCase):
+
+    def setUp(self):
+        self.client.force_login(self.get_superuser())
+        self.menu_content = factories.MenuContentWithVersionFactory()
+        factories.ChildMenuItemFactory.create_batch(5, parent=self.menu_content.root)
+        self.preview_url = reverse(
+            "admin:djangocms_navigation_menuitem_preview", args=(self.menu_content.id,)
+        )
+
+    def test_invalid_menucontent_id_raises_404(self):
+        """
+        Check that 404 response is returned if the menu_content_id is invalid
+        """
+        url = reverse("admin:djangocms_navigation_menuitem_preview", args=(999,))
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 404)
+
+    def test_add_item_link_not_present(self):
+        """
+        Check the button to add an item is not present
+        """
+        response = self.client.get(self.preview_url)
+        add_url = reverse(
+            "admin:djangocms_navigation_menuitem_add", args=(self.menu_content.id,)
+        )
+
+        self.assertEqual(response.status_code, 200)
+        self.assertNotContains(response, "Add menu item")
+        self.assertNotContains(response, add_url)
+
+    def test_versions_link_not_present(self):
+        """
+        Check the button to view versions is not present
+        """
+        response = self.client.get(self.preview_url)
+        version_url = version_list_url(self.menu_content)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertNotContains(response, "Versions")
+        self.assertNotContains(response, version_url)
+
+    def test_actions_not_present(self):
+        """
+        Check that the Actions column is not present
+        """
+        response = self.client.get(self.preview_url)
```

Comment:
Could you add a new line after this, so that the format is consistent with the other tests?